web/package.json

```diff
@@ -29,9 +29,9 @@
     "start:test": "cross-env REACT_APP_ENV=test MOCK=none umi dev",
     "test:component": "umi test ./src/components",
     "tsc": "tsc",
-    "cypress:open": "CYPRESS_SERVE_ENV=test cypress open",
-    "cypress:open-dev": "CYPRESS_SERVE_ENV=dev cypress open",
-    "cypress:run-ci": "CYPRESS_SERVE_ENV=test cypress run"
+    "cypress:open": "cross-env CYPRESS_SERVE_ENV=test cypress open",
```

Comment:
`cross-env` is used in Windows IMO 🤔 we can't set it here directly.

Reply:
Maybe a FAQ or some tips would be better?

Reply:
But If we don't use the `cross-env` to set the env variables, the cypress definitely will fail to start.

Reply:
fail on windows, right? does this work on Linux?

Reply:
Fail on Windows, working in Linux successfully

Reply:
ok, cc @LiteSun to take a look., I remember he had faced this case.

Reply:
there are many ways to set env, see: https://docs.cypress.io/guides/guides/environment-variables.html#Option-4-env , and  `cross-env` is ok IMO🤔